assignment/src/com/assignmentweek2/BinarySearch/Main.java

```diff
@@ -0,0 +1,51 @@
+package com.assignmentweek2.BinarySearch;
+
+import java.util.Scanner;
+import java.util.TreeSet;
+/*
+This is the main method
+ */
+public class Main {
+    public static void main(String[] args) {
+        TreeSet <Number> elementSet =new TreeSet<>();
+        Scanner scanner= new Scanner(System.in);
+        int inputSize = getInputSize(scanner);
+        AddingElements(elementSet, scanner, inputSize);
+        SearchingElement(elementSet, scanner);
```

Comment:
Method names should start with small letters only. Please change in all places wherever it started with Capital letter.

Reply:
changed